### PR TITLE
0.13.x into develop

### PR DESCRIPTION
**Pull request by MCGallaspy**
Opening this after fixing merge conflicts, in order to inspect the changes more closely. Perhaps will give some clues as to #3536.

### CODE REVIEW

File: kalite/version.py [hunk 1]
```diff
@@ -43,13 +43,43 @@
             "all": [],
             "students": [],
             "coaches": [],
+            "admins": ["Language packs will now update properly across KA Lite versions."],
+        },
+
+    },
+
+    "0.13.0": {
+        "release_date": "2015/04/27",
```
Comment (MCGallaspy) [inline kalite/version.py]: I might screwed up this release date.

**Comment by MCGallaspy** [comment]
@rtibbles these changes are small enough to maybe spot the bug my inspection.

**Comment by rtibbles** [comment]
I can't see anything here that might be causal in #3536 .

Is it fixed in this merge?

**Comment by MCGallaspy** [comment]
No, it's still there.

**Comment by MCGallaspy** [comment]
I decided to check out `0.13.x` to make sure I wasn't crazy -- the bug is there, too. Earlier it wasn't, now it is.

**Comment by rtibbles** [comment]
Any errors in the JS console?

**Comment by MCGallaspy** [comment]
Yeah, one. Here's my log:
```
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/exercise.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/content.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/video.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/audio.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/pdf.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/topics.js".
GET http://127.0.0.1:8008/static/js/jquery.qtip.min.js.map 404 (NOT FOUND)
Download the React DevTools for a better development experience: http://fb.me/react-devtools
Uncaught TypeError: Cannot read property 'appendChild' of null
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/exercise.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/content.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/video.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/audio.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/pdf.js".
Resource interpreted as Script but transferred with MIME type text/html: "http://127.0.0.1:8008/handlebars/templates/topics.js".
Each child in an array should have a unique "key" prop. Check the render method of Renderer. See http://fb.me/react-warning-keys for more information.
```
The uncaught type error is from a minified mathjax file.

**Comment by MCGallaspy** [comment]
Could be worthwhile to grab an un-minified version for debugging. I'll check it out.

**Comment by rtibbles** [comment]
I have seen those errors occur on non-problematic pages as well, so may not
be the culprit in the end.

**Pull request closed without merging**